control_freq: 2.0 # [Hz]
goal_reached_dist: 0.2 # [m]
max_path_dist: 1.5 # [m]
look_ahead_dist: 1.00 # [m]
max_velocity: 0.20 # [m/s]
max_angular_rate: 0.5 # [rad/s]
